test/scratchOrg.ts

```diff
@@ -0,0 +1,343 @@
+/*
```

Comment:
scratchOrg was renamed to testSetup.  test/scratchOrg.ts should not be part of your PR.